### PR TITLE
[xc-admin] improve xc-admin-frontend UI/UX

### PR DESCRIPTION
**Pull request by cctdaniel**

### CODE REVIEW

File: governance/xc_admin/packages/xc_admin_frontend/pages/index.tsx [hunk 1]
```diff
@@ -99,44 +144,46 @@ const Home: NextPage<{
...
+              TAB_INFO.UpdatePermissions.queryString ? (
+              <UpdatePermissions />
+            ) : tabInfoArray[currentTabIndex].queryString ===
+              TAB_INFO.Proposals.queryString ? (
+              <Proposals
+                publisherKeyToNameMapping={publisherKeyToNameMapping}
+                multisigSignerKeyToNameMapping={multisigSignerKeyToNameMapping}
+              />
+            ) : null}
+          </StatusFilterProvider>
```
Comment (guibescos) [inline governance/xc_admin/packages/xc_admin_frontend/pages/index.tsx:186]: Why does the general tab need to be inside `StatusFilterProvider`

File: governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx [hunk 2]
```diff
@@ -146,6 +143,7 @@ const General = () => {
 
   // function to download json file
   const handleDownloadJsonButtonClick = () => {
+    console.log(proposeSquads?.wallet.publicKey.toBase58())
```
Comment (guibescos) [inline governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx:146]: console log

File: governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx [hunk 3]
```diff
@@ -714,17 +717,17 @@ const General = () => {
 
   // create anchor wallet when connected
   useEffect(() => {
-    if (connected) {
+    if (connected && proposeSquads) {
       const provider = new AnchorProvider(
         connection,
-        anchorWallet as Wallet,
+        proposeSquads?.wallet,
```
Comment (guibescos) [inline governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx:723]: mmm I don't think you need this `?`

File: governance/xc_admin/packages/xc_admin_frontend/components/tabs/UpdatePermissions.tsx [hunk 4]
```diff
@@ -334,17 +332,17 @@ const UpdatePermissions = () => {
 
   // create anchor wallet when connected
   useEffect(() => {
-    if (connected) {
+    if (connected && proposeSquads) {
       const provider = new AnchorProvider(
         connection,
-        anchorWallet as Wallet,
+        proposeSquads?.wallet,
```
Comment (guibescos) [inline governance/xc_admin/packages/xc_admin_frontend/components/tabs/UpdatePermissions.tsx:338]: No need for `?`

File: governance/xc_admin/packages/xc_admin_frontend/pages/index.tsx [hunk 5]
```diff
@@ -60,11 +93,23 @@ const TAB_INFO = {
...
+}> = ({
+  OPS_WALLET,
+  publisherKeyToNameMapping,
+  multisigSignerKeyToNameMapping,
+}) => {
   const [currentTabIndex, setCurrentTabIndex] = useState(0)
   const tabInfoArray = Object.values(TAB_INFO)
+  const anchorWallet = useAnchorWallet()
+  const wallet =
+    OPS_WALLET !== null
```
Comment (guibescos) [inline governance/xc_admin/packages/xc_admin_frontend/pages/index.tsx:108]: `OPS_WALLET ? (new NodeWallet(
          Keypair.fromSecretKey(Uint8Array.from(OPS_WALLET))
        ) as Wallet)` seems better style

**Review by guibescos — APPROVED** [review]
I feel like there are too many changes but lgtm.

**Review by guibescos — CHANGES_REQUESTED** [review]
Can you fix vercel preview

**Review by guibescos — COMMENTED** [review]
Typescript comments dropped

File: governance/xc_admin/packages/xc_admin_frontend/pages/index.tsx [hunk 6]
```diff
@@ -60,11 +93,23 @@ const TAB_INFO = {
...
+      ? (new NodeWallet(
+          Keypair.fromSecretKey(Uint8Array.from(OPS_WALLET))
+        ) as Wallet)
```
[the remark this answers is not in the document]
Comment (cctdaniel) [reply governance/xc_admin/packages/xc_admin_frontend/pages/index.tsx:111]: yes

**Pull request merged**